routevar/repo.go

```diff
@@ -20,8 +20,26 @@ var (
 // FixRepoRevVars is a mux.PostMatchFunc that cleans and normalizes
 // the route vars pertaining to a RepoRev.
 func FixRepoRevVars(req *http.Request, match *mux.RouteMatch, r *mux.Route) {
+	if _, present := match.Vars["ResolvedRev"]; present {
+		match.Vars["ResolvedRev"] = strings.TrimPrefix(match.Vars["ResolvedRev"], "@")
+	}
+	FixResolvedRevVars(req, match, r)
+}
+
+// PrepareRepoRevRouteVars is a mux.BuildVarsFunc that converts from a
+// RepoRevSpec's route vars to components used to generate routes.
+func PrepareRepoRevRouteVars(vars map[string]string) map[string]string {
+	vars = PrepareResolvedRevRouteVars(vars)
+	if vars["ResolvedRev"] != "" {
+		vars["ResolvedRev"] = "@" + vars["ResolvedRev"]
```

Comment:
is this line correct? this would set the variable to `{resolved-rev}@{resolved-rev}`, right?